Nuevo pull request para mergeo de cambios de la página del detalle

raquelrh opened the pull request:
Estos son los cambios que se llevarán a la rama principal:

1. - Se ha modificado el texto del artículo para que se adapte al artículo 5 definido en el main.html.  
2. - También he cambiado la foto del detalle del artículo acorde al texto.
3. - He validado la página con el w3c.

raquelrh committed:
Actualización de la estrucutura del fichero detail

Se ha validado la estructura del fichero con validator.w3.org para que vaya cumpliendo con los requisitos.

raquelrh committed:
Merge branch 'master' into feature-html-detail

raquelrh committed:
Merge branch 'master' into feature-html-detail

raquelrh committed:
Cambios en el texto y la imagen del artículo

Se añade la categoría, autor y tema acorde a la página principal. Se modifica la imágen y el contenido del artículo

raquelrh committed:
Actualización del documento

Se actualiza el detalle después de validarlo

raquelrh committed:
Cambio en el nombre de la referencia del perfil

Se modifica el nombre del html que contiene el perfil

osperi reviewed (changes requested):
Tefalta por poner la tag ul

raquelrh committed:
Se añade la tag ul

Se añaden las tag li y ul en los menús de navegación

raquelrh commented:
He subido de nuevo el fichero con la tag ul.

raquelrh commented:
He añadido la tag ul además de modificar el nombre del fichero de "Mi perfil"

raquelrh commented:
- Se ha añadido la tag ul 
- Se ha modificado el nombre del fichero de "Mi perfil".
- Se han modificado las referencias de los href para la navegación
- Se modifica el href del "Nuevo Artículo"

raquelrh committed:
Modificación de enlace del logo y de páginas principal y perfil

Se modifican los href de los enlaces para la navegación a la página principal y a la página del perfil

raquelrh committed:
Se modifica el nombre de href del nuevo artículo

Se cambia NewArtivle por NewArticle

osperi reviewed (approved):
Me parece correcto
